### PR TITLE
adding ginkgo and gomega table driven test [WIP]

### PR DESCRIPTION
**Pull request by OmerBenHayun**

### CODE REVIEW

File: pkg/json/json_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+		`"_named_uuid":` + expectedNamedUuid + `,` +
+		`"name":"",` +
+		`"nb_cfg":0,` +
+		`"nb_cfg_timestamp":0,` +
+		`"sb_cfg":0,` +
+		`"sb_cfg_timestamp":0,` +
+		`"hv_cfg":0,` +
+		`"hv_cfg_timestamp":0,` +
+		`"external_ids":["map",[]],` +
+		`"connections":["uuid","71a565df-188f-42d0-9f18-74e180e27889"],` +
```
Comment (roytman) [inline pkg/json/json_suite_test.go]: connections should be type of set, please add an additional entry

**Review by roytman — CHANGES_REQUESTED** [review]

**Comment by hunchback** [comment]
would like to add my input that ginkgo test style has many disadvantages, and thus is not used in many leading project ... it in fact brings to very large functions making things less maintainable. I would suggest we refrain from going ahead with this change ... i can supply examples if needed

**Comment by roytman** [comment]
Hi @hunchback , please provide this input about gingo/gomega. Maybe not in this PR, but as a separate file/note.

**Comment by hunchback** [comment]
closing (can be reopened if becomes relevant)

**Pull request closed without merging**